session_rpcserver.go

```diff
@@ -1219,13 +1227,17 @@ func (s *sessionRpcServer) marshalRPCSession(sess *session.Session) (
 			for feature, rules := range info.FeatureRules {
 				ruleMap := make(map[string]*litrpc.RuleValue)
 				for name, rule := range rules {
-					val, err := s.cfg.ruleMgrs.InitRuleValues(name, []byte(rule))
+					val, err := initRuleValues(
```

Comment:
Just a question so I understand for my own future PRs, when is it ok to add these types of just code reformatting changes that isn't strictly related to the commit they're included to into a commit that's touching related code, and when should they be added to a separate commit? Is it mainly if it's in the same file and it's just 1-2 lines of code fixes, or where do we normally draw the line :)?

Reply:
Ok thanks! Will keep this as the guideline going forward :)!